action.yml

```diff
@@ -0,0 +1,35 @@
+
+# action.yml
+name: "Lazy Action Nodejs"
+description: "Github action to perform build, test , scan and generate image."
+inputs:
+  dockerfile_dir_path:
+    description: "Directory path to the dockerfile"
+    required: true
+  npm_test_placeholder:
+    description: "npm test run placeholder"
+    required: false
+    default: "test"
+  sonar_scan_in_docker:
+    description: "Is sonar scan running as part of Dockerfile"
+    required: false
+    default: "false"
+  sonar_scan_in_docker_target:
+    description: "sonar scan in docker target"
+    required: false
+    default: "sonarscan-env"
+  github_token:
```

Comment:
Is this required? If so, then where is it used?

Reply:
I have added this change thinking they might have dockerscan in dockerfile like the way we have given option for lazy action dotnet.

Reply:
If it is not being used, don't add. I don't see it being required in the near future

Reply:
Removed those changes for now .